frontend/src/components/InterviewPage.js

```diff
@@ -0,0 +1,70 @@
+import { useState, useEffect } from "react"
+import { Link, useNavigate, useLocation } from "react-router-dom"
+import NavBar from './NavBar.js'
+import { Button, Modal } from 'react-bootstrap'
+import ReactMarkdown from "react-markdown"
+import MatchingService from "../services/matchingService.js"
+
+
+const InterviewPage = (event) => {
+    const [questionTitle, setQuestionTitle] = useState("")
+    const [questionDescription, setQuestionDescription] = useState("")
+    const [showEndInterview, setShowEndInterview] = useState(false)
+
+    const location = useLocation()
+    const navigate = useNavigate()
+
+    const interviewId = location.state.interviewId
+
+    useEffect(() => {
+        MatchingService
+            .getInterview(interviewId)
+            .then(interviewDetails => {
+                console.log(interviewDetails)
+                setQuestionTitle(interviewDetails.data.question.title)
+                setQuestionDescription(interviewDetails.data.question.description)
+            })
+    }, [])
+
+    const handleCloseEndInterview = () => setShowEndInterview(false)
+    const handleShowEndInterview = () => {
+        setShowEndInterview(true)
+    }
+    const handleEndFindMatch = () => {
```

Comment:
Suggest to have 1 line spacing

Reply:
Reason for no 1 line spacing is because this 3 functions are grouped together and required for the modal. Grouping together would be clearer to show their purpose together